Dockerfile

```diff
@@ -14,17 +14,17 @@ RUN ./build.sh
 # Final runtime stage.
 FROM alpine
 
-RUN mkdir /plugins
-COPY --from=builder /opt/bin/plugin_antispam_ocr /plugins/
-COPY --from=builder /opt/bin/media_repo /opt/bin/import_synapse /opt/bin/export_synapse_for_import /opt/bin/gdpr_export /opt/bin/gdpr_import /opt/bin/s3_consistency_check /usr/local/bin/
-
 RUN apk add --no-cache \
         su-exec \
         ca-certificates \
         dos2unix \
         imagemagick \
         ffmpeg
 
+RUN mkdir /plugins
```

Comment:
Move `apk add` before copy from builder for better docker build layer reuse.

Reply:
I somewhat recall putting the order like this on purpose - could this change be split into a dedicated PR to prevent this PR from being blocked behind figuring out why?